src/test/java/org/ahocorasick/trie/PayloadTrieTest.java

```diff
@@ -461,6 +462,24 @@ public void testWholeWords() {
         Collection<PayloadEmit<String>> result2 = new LinkedList<>();
         trie.parseText("foobar", result2::add);
 
+        assertTrue(result1.isEmpty());
+        assertEquals(result1, result2);
+    }
+
+    // @see https://github.com/robert-bor/aho-corasick/issues/85
```

Comment:
A couple of items of note:

1. Prefer to not reference GitHub issues (if we migrate to a different host, the comment goes stale).
1. Prefer to use `/** ... */` style comment for methods; it's okay to paste the relevant parts of the issue into the source code comment.

Reply:
same thing as below, I saw it's pretty common in this unit test class to add links to the related issues. I think it's a good for reference.